calico/common.py

```diff
@@ -622,6 +703,29 @@ def validate_tags(profile_id, tags):
         raise ValidationFailed(" ".join(issues))
 
 
+def validate_labels(profile_id, labels):
+    issues = []
+
+    if not VALID_ID_RE.match(profile_id):
+        issues.append("Invalid profile_id '%r'." % profile_id)
+    _validate_label_dict(issues, labels)
```

Comment:
Is issues being passed by reference here?  In several languages I wouldn't expect this to work - not sure about Python!